loom/engine/loom2d/l2dScript.cpp

```diff
@@ -319,6 +321,8 @@ static int registerLoom2D(lua_State *L)
        .addConstructor<void (*)(void)>()
        .addProperty("nativeTextureID", &Quad::getNativeTextureID, &Quad::setNativeTextureID)
        .addProperty("nativeVertexDataInvalid", &Quad::getNativeVertexDataInvalid, &Quad::setNativeVertexDataInvalid)
+       .addMethod("getShader", &Quad::getShader)
```

Comment:
On the outside it is. Compiler was giving me some problems, so this method is called from script. I'll revisit this.